buildspec/packageTestVsix.yml

```diff
@@ -20,6 +21,9 @@ phases:
     pre_build:
         commands:
             - export HOME=/home/codebuild-user
+            # Set up VSCODE_RIPGREP_TOKEN for GitHub API access
+            - export VSCODE_RIPGREP_TOKEN=${GITHUB_READONLY_TOKEN}
+            - echo "Setting up VSCODE_RIPGREP_TOKEN for GitHub API access"
```

Comment:
why do we need this?

Reply:
similar as the GITHUB TOKEN env, set it setting this up to increase GitHub API Limit while download the ripgrep binary. otherwise, the ripgrep download failed.